### ISSUE TITLE
Conflict with native props using @ember-decorators/argument

### ISSUE THREAD

**Issue by lifeart** [toranb/ember-cli-hot-loader#96]
`@ember-decorators/argument` has rule https://github.com/ember-decorators/argument#ignorecomponentswithoutvalidations.

Bu default it's enabled, and this is cause this issue with hot-loader

```Assertion Failed: Attempted to assign the argument 'target' on an instance of UiLoadingSpinner, but no argument was defined for that key. Use the @argument helper on the class field to define an argument for that class.```

No target attr on compnent
```js
import Component from '@ember/component';

export default class UiLoadingSpinner extends Component.extend({
  // anything which *must* be merged to prototype here
}) {
  // normal class body definition here
};

```

I'm wondering, can we check is property target exists under component or not in https://github.com/toranb/ember-cli-hot-loader/blob/master/addon/components/hot-replacement-component.js#L113

**Comment by lifeart** [comment]
fixed in `@ember-decorators/argument` > v0.8.19

**Comment by lifeart** [comment]
Published in `@ember-decorators/argument` - `0.8.20`
